docs/opentelemetry-zio-logging.md

```diff
@@ -0,0 +1,144 @@
+---
+id: opentelemetry-zio-logging
+title: "OpenTelemetry ZIO Logging"
+---
+
+`zio-opentelemetry` logging facilities are implemented around OpenTelemetry Logging.
+
+In order to use `zio-opentelemetry` feature with `zio-logging` you should use `zio-opentelemetry-zio-logging` module.
+
+`OpenTelemetry ZIO Logging` contains utilities for combining ZIO Opentelemetry with ZIO Logging
+
+## Installation
+
+```scala
+"dev.zio" %% "zio-opentelemetry-zio-logging" % "<version>"
+```
+
+## Features
+
+### Log formats
+
+This library implements [Log Format](https://zio.dev/zio-logging/formatting-log-records) for span information (`spanId` and `traceId`). 
+To use them you need a `LogFormats` service in the environment. For this, use the `ZioLogging.logFormats` layer which in turn required a suitable `ContextStorage` implementation.
+
+```scala
+//> using scala "2.13.14"
+//> using dep dev.zio::zio:2.1.1
+//> using dep dev.zio::zio-opentelemetry:3.0.0-RC23+20-a55ed40c+20240530-2226-SNAPSHOT
+//> using dep dev.zio::zio-opentelemetry-zio-logging:3.0.0-RC23+20-a55ed40c+20240530-2226-SNAPSHOT
```

Comment:
The last thing. Please change the version to `3.0.0-RC24`

Reply:
Fixed